<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>496</width>
    <height>294</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Удаление события</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>20</number>
    </property>
    <property name="topMargin">
     <number>20</number>
    </property>
    <property name="rightMargin">
     <number>20</number>
    </property>
    <property name="bottomMargin">
     <number>20</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Yu Gothic UI</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Введите номер удаляемого события:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="input_item_index">
      <property name="font">
       <font>
        <family>Yu Gothic UI</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 10px;</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="delete_event">
      <property name="font">
       <font>
        <family>Yu Gothic UI</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background: rgb(32, 255, 255);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: rgb(0, 192, 255);
}</string>
      </property>
      <property name="text">
       <string>Удалить событие</string>
      </property>
      <property name="autoRepeat">
       <bool>false</bool>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="delete_all_events">
      <property name="font">
       <font>
        <family>Yu Gothic UI</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background:rgb(246, 76, 76);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: rgb(147, 27, 28);
}</string>
      </property>
      <property name="text">
       <string>Удалить все события</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
